lib/nexmo_developer/app/assets/javascripts/digitalData.js

```diff
@@ -1,3 +1,5 @@
+const PRIMARY_CATEGORY = 'biz:api:adp:developer'
```

Comment:
This should also be an ENV variable.

Reply:
This constant variable is used only in this file. If we have to have it as an ENV variable, we need to set the variable in a hidden field tag in the html.erb file and read it from there using javascript. Please advise.

Reply:
remember that station is used by other sites so we should not have hardcoded values

Reply:
I removed the hardcoded value and changed to ENV variable.